samples/snippets/csharp/new-in-7/patternmatch.cs

```diff
@@ -110,7 +110,7 @@ public static int DiceSum5(IEnumerable<object> values)
                         sum += val;
                         break;
                     case PercentileDie die:
-                        sum += die.Multiplier * die.Value;
+                        sum += die.TensDigit * 10 + die.OnesDigit;
```

Comment:
given that you're saying that one of the dice are labeled 00, 10, 20, 90, etc., you shouldn't have to multiply it here. Or you need to change the text to make both dice be only digits, no?

Reply:
I updated it and clarified the markdown.